src/dialects/abstract/query-interface-typescript.ts

```diff
@@ -12,16 +14,99 @@ export class AbstractQueryInterfaceTypeScript {
   }
 
   /**
-   * Creates a new database schema.
+   * Create a new database schema.
    *
    * **Note:** this is a schema in the [postgres sense of the word](http://www.postgresql.org/docs/9.1/static/ddl-schemas.html),
    * not a database table. In mysql and sqlite, this command will do nothing.
    *
-   * @param schema
-   * @param options
+   * @see
+   * {@link Model.schema}
+   *
+   * @param schema Name of the schema
+   * @param [options={}] CreateSchemaQueryOptions
+   * @param [options.collate=null]
+   * @param [options.charset=null]
+    *
+   * @returns
    */
   async createSchema(schema: string, options?: CreateSchemaOptions): Promise<void> {
     const sql = this.queryGenerator.createSchemaQuery(schema, options);
     await this.sequelize.queryRaw(sql, options);
   }
+
+  /**
+   * Drop a single schema
+   *
+   * **Note:** this is a schema in the [postgres sense of the word](http://www.postgresql.org/docs/9.1/static/ddl-schemas.html),
+   * not a database table. In mysql and sqlite, this drop a table matching the schema name
+   *
+   * @param schema Name of the schema
+   * @param [options={}] query options
+   * @param [options.logging] A function that logs sql queries, or false for no logging
+   *
+   * @returns
+   */
+  async dropSchema(schema: string, options?: QueryRawOptions): Promise<void> {
+    const dropSchemaQuery: string | QueryGeneratorDropSchemaQueryObject = this.queryGenerator.dropSchemaQuery(schema);
+
+    let sql: string;
+    let queryRawOptions = options;
+    if (typeof dropSchemaQuery === 'string') {
+      sql = dropSchemaQuery;
+    } else {
+      sql = dropSchemaQuery.query;
+
+      // QueryRawOptions doesn't take undefined bind
+      const bind = dropSchemaQuery.bind as BindOrReplacements;
+      queryRawOptions = { ...options, bind };
+    }
+
+    await this.sequelize.queryRaw(sql, queryRawOptions);
+  }
+
+  /**
+   * Drop all schemas.
+   *
+   * **Note:** this is a schema in the [postgres sense of the word](http://www.postgresql.org/docs/9.1/static/ddl-schemas.html),
+   * not a database table. In mysql and sqlite, this is the equivalent of drop all tables.
```

Comment:
Is the documented behaviour for sqlite correct? We're not testing on that since it does not support schemas